docs/02-guides/display-a-map.mdx

```diff
@@ -0,0 +1,104 @@
+| defaultBounds   | The default map bounds                   | [`CoordinatesShape[]`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/784684ce56cca69ca5c2e42d5d421a8c0b4bb9c3/src/web/theme/components/organisms/Map/location.js#L59-62)   |
+| defaultCenter   | The default center for the map.          | [`CoordinatesShape` ](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/784684ce56cca69ca5c2e42d5d421a8c0b4bb9c3/src/web/theme/components/organisms/Map/location.js#L59-62)    |
+| onBoundsChanged | Event handler for bound changes          | `(event) => void`                                                                                                                                                                                 |
+
+#### Marker Component
+
+| Property      | Description                            | Type                                                                                                                                                                                            |
+| ------------- | -------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| location      | Location for the marker                | [`LocationInputShape`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/784684ce56cca69ca5c2e42d5d421a8c0b4bb9c3/src/web/theme/components/organisms/Map/location.js#L65-74) |
+| icon          | Allows you to override the marker icon | [`object`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/784684ce56cca69ca5c2e42d5d421a8c0b4bb9c3/src/web/theme/components/organisms/Map/index.js#L25-31)                |
+| isPopupOpened | Controlled method for marker popup     | `boolean`                                                                                                                                                                                       |
+| onClick       | Event handler on marker click          | `(event) => void`                                                                                                                                                                               |
+
+### Install a map extension
+
+Follow the installation instruction of the map extension you want to use:
+
+- [Open Street Map (OSM) with Leaflet](/docs/extensions/leaflet#installation)
+- [Google Maps](/docs/extensions/google-maps#installation)
+
+After this, you should be able to use the Map component. See
+[Map.story.js](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/784684ce56cca69ca5c2e42d5d421a8c0b4bb9c3/modules/map-googlemap/web/theme/components/organisms/Map/Map.story.js)
```

Comment:
should this not point to a new storybook story? `Map.stories.tsx`

Reply:
All those links will have to be updated after the related MRs are merged anyway, consider those as temporary.

Reply:
why not link to the referenced MR instead? you're above link is already a perma link (eg not targeting main) so linking to the MR, will already have the future version as a perma link, eg https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/311460ceda02a12fb15f575b29ae612ee34346cb/packages/google-maps/theme/components/Map.stories.tsx

Reply:
Because linking the MR in the current state isn't right until it has been merged.
Your permalink is pointing to a specific commit that may change if / when I rebase the MR to fix the code based on reviews.

Reply:
Updated all links